src/databricks/labs/ucx/hive_metastore/table_move.py

```diff
@@ -0,0 +1,292 @@
+from functools import partial
+
+from databricks.labs.blueprint.installation import Installation
+from databricks.labs.blueprint.parallel import Threads
+from databricks.labs.lsql.backends import SqlBackend, StatementExecutionBackend
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.errors import NotFound
+from databricks.sdk.service.catalog import (
+    PermissionsChange,
+    Privilege,
+    SecurableType,
+    TableType,
+)
+
+from databricks.labs.ucx.config import WorkspaceConfig
+from databricks.labs.ucx.framework.utils import escape_sql_identifier
+from databricks.labs.ucx.hive_metastore.table_migrate import logger
+
+
+class TableMove:
+    def __init__(self, ws: WorkspaceClient, backend: SqlBackend):
+        self._backend = backend
+        self._ws = ws
+
+    @classmethod
+    def for_cli(cls, ws: WorkspaceClient, product='ucx'):
+        installation = Installation.current(ws, product)
+        config = installation.load(WorkspaceConfig)
+        sql_backend = StatementExecutionBackend(ws, config.warehouse_id)
+        return cls(ws, sql_backend)
+
+    def move_tables(
```

Comment:
```suggestion
    def move(
```
Tables suffix is redundant